worker/caasunitprovisioner/deployment_worker.go

```diff
@@ -38,7 +37,7 @@ func newDeploymentWorker(
 ) (worker.Worker, error) {
 	w := &deploymentWorker{
 		application:        application,
-		jujuManagedUnits:   jujuManagedUnits,
```

Comment:
This is deleting the setting of the attribute but there's all the business logic in the worker loop as well